### PR TITLE
Add release notes for v0.11.0

### PR DESCRIPTION
**Pull request by Jaanki**
Depends on https://github.com/submariner-io/releases/issues/241

Signed-Off-By: Janki Chhatbar <jchhatba@redhat.com>

### CODE REVIEW

File: src/content/community/releases/_index.en.md [hunk 1]
```diff
@@ -4,6 +4,11 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.11.0-rc0
+
+* `subctl cloud prepare` command now supports GCP as well as generic kubernetes clusters.
+* `--ignore-requirements` flag is added to `subctl join` command which ignores Submariner requirement checks.
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md]:
```suggestion
* `--ignore-requirements` flag was added to `subctl join` command which ignores Submariner requirement checks.
```

**Review by tpantelis — CHANGES_REQUESTED** [review]

File: src/content/community/releases/_index.en.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.11.0-rc0
```
Comment (tpantelis) [inline src/content/community/releases/_index.en.md]:
```suggestion
## v0.11.0
```

File: src/content/community/releases/_index.en.md [hunk 3]
```diff
@@ -4,6 +4,11 @@ title = "Releases"
...
+
+* `subctl cloud prepare` command now supports GCP as well as generic kubernetes clusters.
```
Comment (skitt) [inline src/content/community/releases/_index.en.md]:
```suggestion
* `subctl cloud prepare` command now supports Google Cloud Platform as well as generic Kubernetes clusters.
```

**Review by skitt — CHANGES_REQUESTED** [review]

**Review by nyechiel — APPROVED** [review]

**Comment by nyechiel** [comment]
@Jaanki can you fix DCO here?

**Review by tpantelis — APPROVED** [review]

**Comment by Jaanki** [comment]
> @Jaanki can you fix DCO here?

@nyechiel DOne

**Review by nyechiel — APPROVED** [review]

**Review by sridhargaddam — APPROVED** [review]

**Review by skitt — APPROVED** [review]

**Pull request merged**